Misc/NEWS.d/next/Core and Builtins/2020-02-18-01-40-13.bpo-39382.OLSJu9.rst

```diff
@@ -0,0 +1,3 @@
+Hold reference to ``__bases__`` tuple item before releasing the tuple itself
+in ``abstract_issubclass()``, avoiding a use-after-free in the next loop
```

Comment:
`abstract_issubclass()` is an implementation detail. Python users do not know anything about it. Use things which expose the bug on the Python level: `issubclass()`.

Also "hold reference to `__bases__` tuple item" says how the bug has been fixed. Instead describe what the bug looked itself from the point of view of the common Python user (a crash in `issubclass()` when `__bases__` contains the only reference to its item, or something like).

Reply:
Gotcha, will change it to a more user-facing message.